app/views/citizens/information/show.html.erb

```diff
@@ -4,11 +4,7 @@
   <h2 class="govuk-heading-m"><%= t('.list.title') %></h2>
 
   <p class="govuk-body">
-    <ul class="govuk-list govuk-list--bullet">
-      <li> <%= t('.list.item_1') %> </li>
-      <li> <%= t('.list.item_2') %> </li>
-      <li> <%= t('.list.item_3') %> </li>
-    </ul>
+    <%= render partial: 'shared/forms/list_items', locals: { translation_path: 'citizens.information.show.list' } %>
```

Comment:
Could you please wrap this up in a helper method (in `ApplicantionHelper` for example). Something like:
```ruby
def list_from_translation_path(translation_path)
  render 'shared/forms/list_items', translation_path: translation_path
end
```
Then the code in the views will be a lot easier to read. For example here it would be:
```erb
<%= list_from_translation_path 'citizens.information.show.list' %>
```